elements/lisk-p2p/src/peer.ts

```diff
@@ -36,14 +36,29 @@ import { P2PRequest } from './p2p_request';
 
 import * as socketClusterClient from 'socketcluster-client';
 import { SCServerSocket } from 'socketcluster-server';
+import { DEFAULT_WS_MAX_PAYLOAD } from './p2p';
```

Comment:
Maybe we can avoid this import by using the incoming payload value in PeerPool because we are instantiating Peer object only their and we can use whatever value is flowing from P2PConfig